chainio/clients/builder.go

```diff
@@ -61,11 +60,11 @@ func BuildAll(config BuildAllConfig, signer signerv2.SignerFn, logger logging.Lo
 		return nil, err
 	}
 
-	txMgr := txmgr.NewSimpleTxManager(ethHttpClient, logger, signer, gethcommon.Address{})
+	// TODO(madhur): is it fine to not set the sender address?
+	txMgr := txmgr.NewSimpleTxManager(ethHttpClient, logger, signerFn, signerAddr)
```

Comment:
right now it's fine. but i suspect when using remote signer it will be needed. as remote signer needs to know that. but leave it for now. right now I don't know what remote signer needs.

Reply:
[b8dc05a](https://github.com/Layr-Labs/eigensdk-go/pull/91/commits/b8dc05a1226dd02fe04b6d4cd0d797e32c206ff7)
oops forgot to remove the TODO. actually this caused a bunch of bugs, the signerAddr has to be set it seems. So I fixed them everywhere, just forgot to remove the TODO. Let me know if you see any others that I forgot to update.